lambdas/db-migration/src/migrations/20200901060125_create_files_granules.ts

```diff
@@ -0,0 +1,48 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => knex.schema
+  .createTable('granules', (table) => {
+    table.bigIncrements('cumulusId').comment('Internal Cumulus ID for a granule').primary();
+    table.text('granuleId').comment('Granule ID');
+    table.text('pdrName').comment('PDR associated with the granule');
+    table.integer('collectionCumulusId').references('cumulusId').inTable('collections').notNullable();
+    table.enum('status', ['running', 'completed', 'failed']).comment('Ingest status of the granule');
+    table.text('execution').comment('Step Function Execution link');
+    table.text('cmrLink').comment('Link to granule in the CMR API');
+    table.text('published').comment('Flag that shows if the granule has been published in CMR');
```

Comment:
I'm not sure what we store in this field, but could/should this be a `boolean`?

Reply:
Yes, https://bugs.earthdata.nasa.gov/browse/CUMULUS-2120 per the schema in d5b910c93

Reply:
It looks like you named the field `boolean` rather than naming it `published` with a type of `boolean`

Reply:
🤦 182b21a57